app.js

```diff
@@ -29,14 +29,14 @@ const data = [{
 }];
 const output = document.querySelector('#portfolio');
 for (let x = 0; x < data.length; x += 1) {
-  output.innerHTML += "<div class='covers'><img class='img__class' src='/img/" + data[x].image + "'><div class='right__box'><h1 class='popup_tekst'>" + data[x].name + "<ul class='selections'><li class='main__txt__wrapper'><a href='#' class='main__txt' id='canopy_1'>CANOPY</a></li><li class='main__txt__wrapper'><a href='#' class='main__txt' id='canopy_2'>FACEBOOK</a></li><li class='circle__wrapper'><a href='#' class='circle'></a></li><li class='list__wrapper'><a href='#' class='list' id='back__1'>BACK END DEV</a></li><li class='list__wrapper'><a href='#' class='list' id='back__2'>Full Stack Dev</a></li><li class='circle__wrapper'><a href='#' class='circle'></a></li><li class='list__wrapper'><a href='#' class='list'>2015</a></li></ul></h1><p class='signup-disc'>" + data[x].description + "<ul class='platform'><li class='pla__btn'>html</li><li class='pla__btn'>css</li><li class='pla__btn'>javascript</li></ul></p><button onclick='openproject(" + x + ")' class='see-project'>see project</button></div></div>";
+  output.innerHTML = `${output.innerHTML}<div class='covers'><img class='img__class' src='/img/${data[x].image}'><div class='right__box'><h1 class='popup_tekst'>${data[x].name}<ul class='selections'><li class='main__txt__wrapper'><a href='#' class='main__txt' id='canopy_1'>CANOPY</a></li><li class='main__txt__wrapper'><a href='#' class='main__txt' id='canopy_2'>FACEBOOK</a></li><li class='circle__wrapper'><a href='#' class='circle'></a></li><li class='list__wrapper'><a href='#' class='list' id='back__1'>BACK END DEV</a></li><li class='list__wrapper'><a href='#' class='list' id='back__2'>Full Stack Dev</a></li><li class='circle__wrapper'><a href='#' class='circle'></a></li><li class='list__wrapper'><a href='#' class='list'>2015</a></li></ul></h1><p class='signup-disc'>${data[x].description}<ul class='platform'><li class='pla__btn'>html</li><li class='pla__btn'>css</li><li class='pla__btn'>javascript</li></ul></p><button onclick='openproject(${x})' class='see-project'>see project</button></div></div>`;
 }
 // const modalTriggerButton = document.querySelector('.see-project');
 const modal = document.querySelector('#popup');
```

Comment:
[ OPTIONAL ]Pop-up for mobile and desktop should follow the design. Pop-up for mobile view has stretched project images and does not show buttons on the bottom. Pop-up for desktop does also not show `See source` and `See live` buttons.
![mobile-pop](https://user-images.githubusercontent.com/35504149/136599189-e868bb71-1039-4290-b627-69dc6d7716fa.png)
![desktop-pop](https://user-images.githubusercontent.com/35504149/136599204-00be738f-38b6-4a6a-ba01-629f33803efc.png)